### ISSUE TITLE
PyPi wheel is might have not been uploaded right

### ISSUE THREAD

**Issue by tcholewik** [dask/dask-cloudprovider#137]
Not sure if this really counts as a bug but PyPi wheel seems to be miss configured.

I tried installing with `pip install dask-cloudprovider[aws]` and I saw azure dependencies being installed(actually failing to install).
On the other hand installing with `pip install git+https://github.com/dask/dask-cloudprovider#egg=dask-cloudprovider[aws]` got the right results.

I got those results while trying to build a docker image so following commands should reproduce this issue:

```
FROM python:rc-slim
RUN apt-get update && apt-get install --yes build-essential libffi-dev git
RUN pip install jupyterlab dask[distributed]
RUN pip3 install dask-cloudprovider[aws]
```

Let me know if you need more info.

**Comment by jacobtomlinson** [comment]
Thanks for raising this @tcholewik.

This is because we haven't released yet since adding the optional dependencies. So for now an install from PyPI will pul in all dependencies. This will work from the next release.

**Comment by jacobtomlinson** [comment]
Resolved as of v0.5.0
